charts/ai-software-templates/pipeline-setup/README.md

```diff
@@ -0,0 +1,51 @@
+# OpenShift Pipelines Application Namespace setup for Devtools AI Sample Applications.
+
+This repo is a Helm chart that a user admin level privileges to his application namespace would run to set up OpenShift Pipelines for the Devtools AI sample applications. For more information about helm charts see the official [Helm Charts Documentation](https://helm.sh/).
+
+
+## Requirements
+
+- You have sufficient permissions to create a namespace.
+- You have sufficient permissions to create RBAC and ServiceAccounts in your namespace.
+
+## Background
+
+This chart assumes the [OpenShift Pipelines install chart](../pipeline-install/README.md) has been run, something equivalent to what it does, so 
+that OpenShift Pipelines can run the Tekton Pipelines under the [rhdh-pipelines](https://github.com/redhat-ai-dev/rhdh-pipelines) repo, and that
+Pipelines As Code has sufficient credentials to process events from you GitHub Application.
+
+This chart will then set up the Quay and Git credentials in Secrets so that the Tekton Pipelines can interact with your
+applications GitHub repository and push you versions of your application's image to your Quay repository.
+
+## Installation
+
+The helm chart can be directly installed from the OpenShift Dev Console. Check [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.8/html/building_applications/working-with-helm-charts#understanding-helm) for more information.
+
+### Install using Helm
+
+To install the Pipelines Setup Helm chart using Helm directly, you can run:
+
+```
+helm upgrade --install <release-name> --namespace openshift-pipelines --create-namespace .
```

Comment:
From the rest of the content I understand the name space here is the application namespace right?

Reply:
yep copy/paste typo ... will fix in next push - thx